CMakeLists.txt

```diff
@@ -152,7 +152,7 @@ include_directories("dlpack/include")
 #  add_subdirectory(dlpack)
 #endif()
 
-if(NOT MSVC)
+if(NOT MSVC AND NOT APPLE)
```

Comment:
clang doesn't support this flag.  Static linking with default flags seems to work properly.